### PR TITLE
fixes/adds/ports mre/snacks/food and literal trash

### PR DESCRIPTION
**Pull request by Alterist**
see #477 
adjusted the MRE menu to make more sense
changes a bunch of sprites

### CODE REVIEW

File: code/game/objects/items/storage/boxes.dm [hunk 1]
```diff
@@ -646,11 +646,9 @@
 		new /obj/item/lightstick/red(src)
 
 
-
-
 /obj/item/storage/box/MRE
 	name = "\improper USCM MRE"
-	desc = "Meal Ready-to-Eat, property of the US Colonial Marines. Meant to be consumed in the field, and has an expiration that is at least two decades past your combat life expectancy."
+	desc = "Meal Ready-to-Eat, meant to be consumed in the field, and has an expiration that is two decades past an marine's average combat life expectancy."
```
Comment (Ghommie) [inline code/game/objects/items/storage/boxes.dm]: "_a marine's_".

File: code/game/objects/items/reagent_containers/food/snacks.dm [hunk 2]
```diff
@@ -2781,15 +2781,15 @@
...
-		if("cracker", "cheese spread", "rice onigiri", "mashed potatoes", "risotto")
+		if("meatballs", "cheese spread", "beef turnover", "mashed potatoes", "yellow crayon" )
 			icon_state = "side"
 			list_reagents = list("nutriment" = 3, "sodiumchloride" = 1)
-		if("biscuit", "meatballs", "pretzels", "peanuts", "sushi")
+		if("biscuit", "pretzels", "peanuts", "cracker", "purple crayon")
 			icon_state = "snack"
 			list_reagents = list("nutriment" = 2, "sodiumchloride" = 1)
-		if("spiced apples", "chocolate brownie", "sugar cookie", "coco bar", "flan", "honey flan")
+		if("spiced apples", "chocolate brownie", "sugar cookie", "choco bar", "blue crayon")
```
Comment (Ghommie) [inline code/game/objects/items/reagent_containers/food/snacks.dm]: I see you removed the chicken tender, risotto, sushi, flan and honey flan, was it intended?

File: code/game/objects/items/reagent_containers/food/snacks.dm [hunk 3]
```diff
@@ -2781,15 +2781,15 @@
 	flavor = newflavor
 
 	switch(newflavor)
-		if("boneless pork ribs", "grilled chicken", "pizza square", "spaghetti chunks", "chicken tender")
+		if("boneless pork ribs", "grilled chicken", "pizza square", "spaghetti", "red crayon")
```
Comment (Qustinnus) [inline code/game/objects/items/reagent_containers/food/snacks.dm]: please do not try to sneak in a chicken tendie removal in your PRs again or I will have to gitban you

Comment (Qustinnus) [reply code/game/objects/items/reagent_containers/food/snacks.dm]: i take my chicken tendies very serious

File: code/game/objects/items/reagent_containers/food/snacks.dm (hunk 2 above)
Comment (Alterist) [reply code/game/objects/items/reagent_containers/food/snacks.dm]: Yes

**Comment by Alterist** [comment]
onigiri and sushi was removed for not fitting thematically in a mre, tendies and risotto were removed for not  having a fitting sprite, same for flan and honey flan, which shared the same sprite

**Comment by Alterist** [comment]
Tendies will be radded after I find a good sprite

File: code/game/objects/items/reagent_containers/food/snacks.dm [hunk 4]
```diff
@@ -2764,7 +2764,7 @@
 /obj/item/reagent_container/food/snacks/packaged_meal/New(loc, newflavor)
 	tastes = list("[pick(food_adjectives)]" = 1) //idea, list, gimmick
 	determinetype(newflavor)
-	desc = "A packaged [icon_state] from a Meal Ready-to-Eat, property of the US Colonial Marines and prepared for field consumption. <i> On the rear is a lengthy list of [pick("obscure", "arcane", "unknowledgeable", "revolutionary", "sophisticated", "unspellable")] ingredients and addictives.</i>"
+	desc = "A packaged [icon_state] from a Meal Ready-to-Eat,on the rear is a lengthy list of [pick("obscure", "arcane", "unknowledgeable", "revolutionary", "sophisticated", "unspellable")] ingredients and addictives.</i>"
```
[the remark this answers is not in the document]
Comment (Alterist) [reply code/game/objects/items/reagent_containers/food/snacks.dm]: this is why i shouldn't touch lore

**Pull request merged**